smurfs/src/reducers/reducer.js

```diff
@@ -0,0 +1,54 @@
+import {FETCH_SMURFS_START, FETCH_SMURFS_SUCCESS, FETCH_SMURFS_FAIL} from "../actions/getSmurfs";
+import {POST_SMURFS_START, POST_SMURFS_SUCCESS, POST_SMURFS_FAIL} from "../actions/postSmurfs";
```

Comment:
creating an `/actions/index.js` where you have a bunch of 
```
export { ACTION_START/SUCCESS/FAILURE } from './actions.js'
``` 
will help clean this up

Reply:
exports: https://github.com/jacobAngulo/Redux-Friends/blob/jacob-angulo/friends/src/actions/index.js

inports: https://github.com/jacobAngulo/Redux-Friends/blob/jacob-angulo/friends/src/reducers/index.js